RedHatInsights/insights-inventory-frontend
# fix(ESSNTL-3760): handle insights disconnected hosts for patch, advis…

Fixes: https://issues.redhat.com/browse/ESSNTL-3760.

This moves insights disconnected systems handling from applications inside the tab to Inventory itself. By having this we have more consistency and we will not be required to load those apps, instead Inventory shows the NotConnected component directly,

To test:
1. Find `ci-vm-10-0-206-95.hosted.upshift.rdu2.redhat.com` host which is of disconnected systems
2. Click and open details
3. Move to Advisor, vuln and Patch tabs
4. Observe that each app is not loaded, instead NotConnected component is shown directly
